src/app/core/pipes/time-format/time-format.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'timeFormat'
+})
+export class TimeFormatPipe implements PipeTransform {
+
+  transform(time: number): string {
+    let formatted = '';
+    const hours = Math.floor(time / 60);
+    const minutes = Math.floor(time) % 60;
+
+    if (hours) {
```

Comment:
I think this would be looks simpler


return (hours)
      ? `${hours}h ${minutes} min`
      : `${minutes} min`;